.travis.yml

```diff
@@ -31,4 +31,6 @@ install:
 
 script:
     - pip install virtualenv==1.10
+    - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install "mercurial>=3.3" ; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install python-hglib==1.5 ; fi
```

Comment:
Does this indicate that the Mercurial support is only on Python 2.x, since `python-hglib` only works there?  Maybe that should be documented?